File: BookClub96/BookClub96/Data/Entities/Meeting.cs
using System;
using System.Collections.Generic;

namespace BookClub96.Data.Entities
{
    public class Meeting
    {
        public int Id { get; set; }
        public Book Book { get; set; }
        public decimal Rating { get; set; }
        public string Address { get; set; }
        public DateTime Time { get; set; }
        public int GroupId { get; set; }

        public ICollection<MeetingMember> Attendees { get; set; }
    }
}

File: BookClub96/BookClub96/ClientApp/app/shared/meeting.ts
import { Book } from "../shared/book"
import { Member } from "../shared/member"

export class Meeting {
    meetingId: number;
    book: Book;
    rating: string;
    address: string;
    groupId: string;
    time: Date;
    attendees: Array<Attendee> = new Array<Attendee>();
    host: Attendee;
}

export class Attendee {
    memberId: string;
    isHost: true;
    member: Member;
}